libs/utils/android/workloads/youtube.py

```diff
@@ -56,6 +56,11 @@ def run(self, exp_dir, video_url, video_duration_s, collect=''):
         # Force screen in LANDSCAPE mode
         Screen.set_orientation(self.target, portrait=False)
 
+        self.target.execute('monkey -p {} 1'.format(self.package))
```

Comment:
These seems more a fix to be posted in a separate patch....
BTW: what is that?!?